### ISSUE TITLE
FTrack: Note creation with empty intent

### ISSUE THREAD

**Issue by gabormarinov** [pypeclub/OpenPype#3138]
**Describe the bug**
`IntegrateFtrackInstance/IntegrateFtrackNote` default Note template:
`{intent}: {comment}`

If the user don't enter any intent or write any comment a Note sill being created in FTrack with content:
`-:`
which is the intent `-`, the separator `:` and an empty string.

**To Reproduce**
1. Switch on `IntegrateFtrackInstance/IntegrateFtrackNote`  in settings with the defaults
2. Publish anything without setting an intent or writing any comment
3. Check the Note in FTrack

**Expected behavior**
If the user don't set an intent or enter a comment, then don't create an FTrack note.

**Screenshots**
Created note example:
![image](https://user-images.githubusercontent.com/8620515/166469483-98288dfa-e047-4f79-ac7c-161cedd499a7.png)

**Comment by mkolar** [comment]
@iLLiCiTiT we should probably make a condition there to skip the note provided both intent and comment are empty

**Comment by iLLiCiTiT** [comment]
The intent is not empty but `-`. We should rather add ability to have "empty intent" so it is not needed to hack it using dash (or different symbol).

**Comment by mkolar** [comment]
I agree, however, then you'll still end up with `:` in the comment won't you?
